.gitignore

```diff
@@ -6,3 +6,9 @@ home/
 
 dump.shm
 dump.shm.lastest.tar.bz2
+
+/vendor
```

Comment:
FYI. I am not aware of this change when creating https://github.com/Ptt-official-app/Ptt-backend/pull/20. Let's be careful when merged.